react/features/base/label/components/styles.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import { BoxModel, ColorPalette, createStyleSheet } from '../../styles';
+
+/**
+ * The styles of the native base/label feature.
+ */
+export default createStyleSheet({
+
+    /**
+     * The outermost view.
+     */
+    indicatorContainer: {
+        alignItems: 'center',
+        backgroundColor: '#808080',
+        borderRadius: 20,
```

Comment:
Why not use a percentage? That way the width can change and maybe this value doesn't. I'm making large assumptions about how border-radius works though.

Reply:
Unfortunately native doesn't support percentage :(

https://facebook.github.io/react-native/docs/view-style-props.html#borderradius